policyengine_us/parameters/gov/states/hi/tax/income/deductions/standard/amount.yaml

```diff
@@ -0,0 +1,21 @@
+description: Hawaii provides filers a standard deduction of this amount, depending on their filing status.
+metadata:
+  label: Hawaii standard deduction
+  period: year
+  unit: currency-USD
+  breakdown:
+  - filing_status
+  reference:
+  - title: 2022 Hawaii Tax Form Instruction Booklet p20
```

Comment:
```suggestion
  - title: 2022 Hawaii Tax Form Instruction Booklet line 23
```

Reply:
Hello Max, thank you for your suggestions. I have already applied these changes and pushed a new commit.